data/apps/reviewer_apps.json

```diff
@@ -0,0 +1,26 @@
+[{"type": "hosted",
+  "status": "pending",
+  "manifest_url": "http://reviewer-app1.testmanifest.com/manifest.webapp",
+  "categories": ["books", "business"],
+  "device_types": ["firefoxos", "desktop"],
+  "privacy_policy": "good privacy policy",
+  "preview_files": ["images/screenshot1.png"],
+  "manifest_file": "manifests/app1.json"
```

Comment:
Yes; I just wanted to set this up to load from a file so we're not dependent on an external resource. Besides, it's easier to customize this way.